schema/check_generated_data.py

```diff
@@ -89,17 +89,17 @@ def main(args):
     except BaseException as error:
         schema_errors.append(output_filename)
         logging.fatal('Error: %s. Cannot save validation summary in file %s', error, output_filename)
-        exit(1)
+        sys.exit(1)
 
     if schema_errors:
         logging.critical('Test data file files: %d fail out of %d:',
                          len(schema_errors), schema_count)
         for failure in schema_errors:
             logging.critical('  %s', failure)
-        exit(1)
+        sys.exit(1)
     else:
         logging.info("All %d generated test data files match with schema", schema_count)
-        exit(0)
+        sys.exit(0)
```

Comment:
Do you need a `sys.exit(0)`?  If this is the last line of the happy case, then it reads better to remove the line. When I see `sys.exit(<nonzero>)`, I assume that someone is signalling an _unnaturally early_ exit, so a really unhappy case. Happy cases and handled unhappy cases would otherwise be treated "normally" (no op, try/catch exception handling, etc.).

Reply:
Fixed.